### PR TITLE
Add Members page

### PR DESCRIPTION
**Pull request by alfakini**
Screen Shot 2021-11-28 at 19 43 18

### CODE REVIEW

**Comment by alfakini** [comment]
[[PAGE] Implement the Members Lists page](https://trello.com/c/Yi1OTiTx/232-page-implement-the-members-lists-page)

File: src/pages/explore/MembersPage/components/PayoutsAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import type { Balance, BlockNumber } from '@polkadot/types/interfaces'
+import { Accordion, Card, useAccordionButton } from 'react-bootstrap'
+import { BlockTime } from '../../../../components/BlockTime'
+import { FormatBalance } from '../../../../components/FormatBalance'
+
+const PayoutsAccordionToggle = ({ children, eventKey }: { children: any, eventKey: any }) => (
+  <a href="javascript:void(0)" onClick={useAccordionButton(eventKey)} >
+    {children}
+  </a>
+)
```
Comment (laurogripa) [inline src/pages/explore/MembersPage/components/PayoutsAccordion.tsx:10]: We won't show payouts on the members list, nor strikea

File: src/pages/explore/MembersPage/MembersList.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <Col xs={2} className="text-start text-truncate">
+          {truncateMiddle(member.accountId?.toString())}
+        </Col>
+        <Col xs={3} className="text-start">
+          {member.hasPayouts ? <PayoutsAccordion payouts={member.payouts} /> : <></>}
+        </Col>
+        <Col xs={6} className="text-end">
+          {member.isFounder ? <Badge pill bg="primary" className="me-2">Founder</Badge> : <></>}
+          {member.isHead ? <Badge pill bg="primary" className="me-2">Head</Badge> : <></>}
+          {member.isDefenderVoter ? <Badge pill bg="primary" className="me-2">Defender</Badge> : <></>}
```
Comment (laurogripa) [inline src/pages/explore/MembersPage/MembersList.tsx:29]: This is confusing as it shows a `defender` tag on members who voted on the defender. This induces the user into thinking that these members are defenders themselves. I think we can remove this.

File: src/pages/explore/MembersPage/MembersList.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          {member.hasStrikes ? <Badge pill bg="danger" className="me-2">{member.strikesCount} Strikes</Badge> : <></>}
```
Comment (laurogripa) [inline src/pages/explore/MembersPage/MembersList.tsx:30]: We can remove the strikes. If needed, we can show it later on member profile.

**Review by laurogripa — CHANGES_REQUESTED** [review]

**Review by laurogripa — APPROVED** [review]

**Review by laurogripa — CHANGES_REQUESTED** [review]
Last commit broke a test case

**Review by danielSbastos — APPROVED** [review]

**Pull request merged**